src/module/addremove.js

```diff
@@ -0,0 +1,88 @@
+const add = () => {
+  const addTask = document.getElementById('addTask'); // get the input field element
+  if (addTask.value !== '') {
+    const tasksArray = JSON.parse(localStorage.getItem('tasksArray')) || [];
+    tasksArray.push({ completed: false, description: addTask.value });
+    for (let i = 1; i <= tasksArray.length; i += 1) {
+      tasksArray[i - 1].index = i;
+    }
+    localStorage.setItem('tasksArray', JSON.stringify(tasksArray));
+  }
+  addTask.value = '';
+};
+
+const remove = (index) => {
+  const tasksArray = JSON.parse(localStorage.getItem('tasksArray')) || [];
+  tasksArray.splice(index, 1);
+  for (let i = 0; i < tasksArray.length; i += 1) {
+    tasksArray[i].index = i + 1;
+  }
+  localStorage.setItem('tasksArray', JSON.stringify(tasksArray));
+};
+
```

Comment:
- [ ] Kindly as stated by the previous reviewer 😃, Good job adding the add function and the edit task function. However, removing a selected task does not work. When the user tries to delete a random task, the task which gets deleted is the task that was added last, so please try to fix that to let the users can benefit from the app💯🚀

- Here is an example: 

```ex
// Assuming you have a parent container for the tasks with an id "taskContainer"

// Attach event listener to the parent container (event delegation)
document.getElementById('taskContainer').addEventListener('click', function(event) {
  if (event.target.classList.contains('deleteButton')) {
    // Get the task element that was clicked
    var taskElement = event.target.parentElement;
    
    // Get the task ID or any other unique identifier from the task element
    var taskId = taskElement.dataset.taskId;
    
    // Find the index of the task in the tasks array
    var taskIndex = tasks.findIndex(function(task) {
      return task.id === taskId; // Assuming each task has an 'id' property
    });
    
    // Remove the task from the tasks array
    if (taskIndex > -1) {
      tasks.splice(taskIndex, 1);
    }
    
    // Remove the task element from the DOM
    taskElement.remove();
  }
});
```

- [ ] You can see here what is happening when I try to remove the task 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
| --- |
|      ![Recording 2023-06-16 at 18 24 19](https://github.com/mohashyne/Todo_list/assets/83360636/d9c2eda3-b2df-426e-a39f-d7b16c170be8)   |